internal/run/plan_file.go

```diff
@@ -13,7 +13,8 @@ const (
 type (
 	// PlanFile represents the schema of a plan file
 	PlanFile struct {
-		ResourceChanges []ResourceChange `json:"resource_changes"`
+		ResourceChanges []ResourceChange  `json:"resource_changes"`
+		OutputChanges   map[string]Change `json:"output_changes"`
```

Comment:
Why a map an not an array `[]OutputChange` like `ResourceChanges`? I would go for consistency.

Reply:
Because that's the schema of the plan file, I have no control over it.